hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -248,10 +291,20 @@ public long getFlushIterations() {
    * @param response
    * @param transactionIndex
    */
-  public synchronized void add(OMClientResponse response,
+  public synchronized CompletableFuture<Void> add(OMClientResponse response,
```

Comment:
Can you return Optional<CompletableFuture<Void>> instead? We should avoid having nulls in code wherever possible.

Reply:
This is for a temporary thing. Sooner, all OM uses HA, so this code will be removed in later point of time. And in HA case, we don't even use future. So, I think this should be okay, let me know you still want to use Optional here?

Reply:
No that's fine. Leave it as it is.